### ISSUE TITLE
English language output is default, with no indication on how to change it

### ISSUE THREAD

**Issue by FredrikKarlssonSpeech** [JuliaNeuralGraphics/Whisper.jl#5]
The transcription system works and provides an output that is content-wise fine, but the language is English always.
Unless I am missing some way of forcing the output language, I suspect that I am not alone in assuming that if I give this command

```
Whisper.transcribe(
           "/Users/frkkan96/Desktop/kaa_yw_pb_16000.flac", "/Users/frkkan96/Desktop/kaa_yw_pb_16000.srt";
           model_name="large-v3", language="swedish", dev=cpu, precision=f32)
```

I would get an .srt file with the Swedish language transcription in it?

**Comment by pxl-th** [comment]
Yes, by default the model performs transcribing to English and if the audio is non-English, then it also translates to english.

To actually transcribe to non-English language you have to specify `language` keyword argument.

To do this automatically, we'd have to run language detection first, before transcribing.

**Comment by FredrikKarlssonSpeech** [comment]
> To actually transcribe to non-English language you have to specify `language` keyword argument.

Sure, and I did (see the post). So, when you want to transcribe, in my case Swedish, the usual case is also to want the output in Swedish language text. Not English language text in the SRT - which is what I got.
